src/diffusers/pipelines/stable_diffusion/README.md

```diff
@@ -72,7 +72,7 @@ image.save("astronaut_rides_horse.png")
 # make sure you're logged in with `huggingface-cli login`
 from diffusers import StableDiffusionPipeline, DDIMScheduler
 
-scheduler = DDIMScheduler(beta_start=0.00085, beta_end=0.012, beta_schedule="scaled_linear", clip_sample=False, set_alpha_to_one=False)
+scheduler =  DDIMScheduler.from_config("CompVis/stable-diffusion-v1-4", subfolder="scheduler")
```

Comment:
[scheduler_config](https://huggingface.co/CompVis/stable-diffusion-v1-4/blob/main/scheduler/scheduler_config.json) does not specify values for `clip_sample` or `set_alpha_to_one`

Reply:
Good point! Wonder if we could somehow do this automatically - maybe just add those values there?

Reply:
Actually opening a bigger issue for this

Reply:
Great catch! Yeah I noticed it's acutally not that easy - we'll need some more code changes here

Reply:
This can be correct now by simply setting `clip_sample` in the original configuration file - even if it's not expected by the scheduler => the original PNDMScheduler config should therefore just have `clip_sample` set to `False` for DDIM. This logic is now implemented via the class attribute `_compatible_classes`.